### PR TITLE
Changing "Sort children by" on folder that contains folder breaks

### PR DESCRIPTION
**Pull request by dturopoli**
Changing "Sort children by" on folder that contains folder breaks causing this error

"Message: Call to undefined method getLatestVersion in class Pimcore\Model\DataObject\Folder"

### CODE REVIEW

**Comment by dvesh3** [comment]
@dturopoli thanks for your PR. Please rebase on branch 10.0, since it is a fix.

**Comment by dturopoli** [comment]
> @dturopoli thanks for your PR. Please rebase on branch 10.0, since it is a fix.

@dvesh3 Changed to 10.0 branch

File: bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php [hunk 1]
```diff
@@ -1184,7 +1184,12 @@ protected function reindexBasedOnSortOrder(DataObject\AbstractObject $parentObje
     private function updateLatestVersionIndex($objectId, $newIndex)
     {
         $object = DataObject\Concrete::getById($objectId);
-        if ($object && $latestVersion = $object->getLatestVersion()) {
+
+        if (
+            $object &&
+            $object->getType() != 'folder' &&
```
Comment (dvesh3) [inline bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php]:
```suggestion
            $object->getType() != DataObject::OBJECT_TYPE_FOLDER &&
```

**Comment by dvesh3** [comment]
@dturopoli thanks!

**Pull request merged**